kotest-framework/kotest-framework-engine/src/jvmTest/kotlin/com/sksamuel/kotest/engine/test/timeout/ContainerTimeoutTest.kt

```diff
@@ -1,30 +1,43 @@
 package com.sksamuel.kotest.engine.test.timeout
 
-import io.kotest.common.ExperimentalKotest
+import io.kotest.assertions.asClue
+import io.kotest.common.testTimeSource
 import io.kotest.core.spec.style.FunSpec
 import io.kotest.engine.TestEngineLauncher
 import io.kotest.engine.listener.CollectingTestEngineListener
+import io.kotest.matchers.collections.shouldContainExactly
+import io.kotest.matchers.nulls.shouldNotBeNull
 import io.kotest.matchers.shouldBe
 import kotlinx.coroutines.delay
 import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.measureTime
 
-@ExperimentalKotest
```

Comment:
I removed `@ExperimentalKotest` because it didn't appear to be necessary. None of the code requires opting-in for `@ExperimentalKotest`. It's a test class, so it won't be published, so no one will ever need to opt-in to use it.

Reply:
sounds good